ste/xferLogPolicy.go

```diff
@@ -98,9 +98,6 @@ func NewRequestLogPolicyFactory(o RequestLogOptions) pipeline.Factory {
 				}
 
 				pipeline.WriteRequestWithResponse(b, prepareRequestForLogging(request), response.Response(), err)
-				if logLevel <= pipeline.LogError {
```

Comment:
We only want to remove the stack trace for HTTP errors. This is removing it altogether.

Reply:
Pardon me, misunderstood the code, thought it was exclusively for HTTP related errors.